index.html

```diff
@@ -100,7 +95,108 @@ <h6>Skills</h6>
+        </p>
+        <section class="language">
+            <h5>css</h5>         
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>html</h5>
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>bootstrap</h5>
+            <img class="line" src="./Rectangle70.png" alt="">
+                <h5>ruby</h5>        
+        </section>
+        <button class="btn" type="button">See Project</button>
+        <img src="./ImgPlaceholder.png" alt="">
+        <h2 class="post-heading">Multi-Post Stories</h2>
+        <p class="post-1">
+            A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <section class="language">
+            <h5>css</h5>         
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>html</h5>
+                <img class="line" src="./Rectangle70.png" alt="">
+            <h5>bootstrap</h5>
+            <img class="line" src="./Rectangle70.png" alt="">
+                <h5>ruby</h5>        
+        </section>
+        <button class="btn" type="button">See Project</button>
+        <h4>CONTACT US</h4>
+        <p class="contact-info">
+            If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form action="#">
+            <input type="text" placeholder="Text Input">
+            <input type="email" placeholder="jeremy@gmail.com">	
+        </form>
+        <textarea name="text" id="input" cols="40" rows="10">Enter text here</textarea>
+        <button class="btn-btn" type="button">See Project</button>
+        <hr>
+        <ul class="social-icons">
+            <li class="icons">
+                <a href="#">
+                    <img src="./assets/git.png" alt="">
+                </a>
+            </li>
+            <li class="icons">
+                <a href="#">
+                    <img src="./assets/ln.png" alt="">
+                </a>
+            </li>
+            <li class="icons">
+                <a href="#">
+                    <img src="./assets/v.png" alt="">
+                </a>
+            </li>
+            <li class="icons">
+                <a href="#">
+                    <img src="./assets/tw.png" alt="">
+                </a>
+            </li>
+            <li class="icons">
+                <a href="">
+                    <img src="./assets/m.png" alt="">
+                </a>
+            </li>
+        </ul>
+        <p class="email">emailhere@email.com</p>
+        <hr class="end-line">            
```

Comment:
- [ ] Kindly let's have a container that wraps the contact section as well, you can use the `<section>` tag since this is a big content.
- [ ] It seems that the paragram `<p class="contact-info">` does not have the same font as the one in the Figma design, see the difference below 👇 

- Figma design
![image](https://user-images.githubusercontent.com/55465133/133836723-00dad7a3-3089-450a-a05f-0c426d271785.png)

- Your design 
![image](https://user-images.githubusercontent.com/55465133/133836783-294c3198-1137-4e7a-a29a-e6a0f352dbe7.png)

Can you see the difference? please let's fix this as well 👍

Reply:
Thank you.